lib/trino-parquet/src/main/java/io/trino/parquet/ParquetTypeUtils.java

```diff
@@ -14,6 +14,7 @@
 package io.trino.parquet;
```

Comment:
You didn't remove `io.trino.parquet.RichColumnDescriptor` class. Is it still used?

Could you improve commit description? Why we don't need `io.trino.parquet.RichColumnDescriptor` anymore

Reply:
Huh I thought I had removed it, it has been removed now. Commit message has been updated as well.